beacon-chain/core/electra/deposits_test.go

```diff
@@ -196,7 +195,7 @@ func TestProcessPendingDeposits(t *testing.T) {
 			},
 		},
 		{
-			name: "process excess balance that uses a point to infinity signature, processed as a topup",
```

Comment:
you could also say ignores signature validation the signature for topup and still use the infinite signature

Reply:
But why though? If the signature field isn't touched, does it matter what signature it uses? It could be empty, junk, or even infinite